_includes/partials/contributor.html

```diff
@@ -13,7 +13,7 @@ <h2>Community Members</h2>
                 <li>
                     <div class="square-image">
                         <a href="{{page.url}}">
-                            <img src="{% include relative-src.html src=page.image_path %}" alt="{{ page.name }}"/>
+                            <img class="b-active" src="{% include relative-src.html src=page.image_path %}" alt="{{ page.name }}"/>
```

Comment:
This won't' work as we are displaying each member by traversing through the md files, and we can't add a class like `b-inactive` beside it in the future as it will make all of the contributors inactive.

Instead, you should make changes to the md file of each member and add a field like status having two values `active` and `inactive` and check for the status in the `contributor.html` file while rendering, using  conditional operator in [Liquid html](https://shopify.github.io/liquid/tags/control-flow/) and then add the class accordingly.

Reply:
Oh yes! extremely sorry about this. I'm working on this!